rhel9/precompiled/Makefile

```diff
@@ -66,9 +66,10 @@ image: rhsm-register
 		--build-arg BUILDER_EMAIL=${BUILDER_EMAIL} \
 		--build-arg DRIVER_TOOLKIT_IMAGE=${DRIVER_TOOLKIT_IMAGE} \
 		--build-arg DRIVER_OPEN=${DRIVER_OPEN} \
-                --build-arg DRIVER_TYPE=${DRIVER_TYPE} \
+        --build-arg DRIVER_TYPE=${DRIVER_TYPE} \
```

Comment:
minor nit: GitHub is showing an indentation issue here (this line is not aligned with the other build args).